subsys/bluetooth/host/settings.c

```diff
@@ -166,6 +178,15 @@ static void generate_static_addr(void)
 	settings_save_one("bt/id", str);
 }
 
+#if CONFIG_BT_DEVICE_NAME_STORAGE > 0
+static void generate_name(void)
```

Comment:
This isn't really generating anything, so the name seems wrong. That said, the function is super short and you're only calling it from one place, so I'd just do the bt_set_name() in line from commit(). Extra bonus is that you get less ugly `#ifdefs` in this file that way as well.

Reply:
Sounds good, that was a bigger function initially but I endup redoing it as bt_set_name.